waku/v2/node/storage/message/message_store.nim

```diff
@@ -6,19 +6,34 @@
 import
   std/options,
   stew/results,
+  chronos
+import
   ../../../protocol/waku_message,
   ../../../protocol/waku_store/rpc,
   ../../../utils/time,
   ../../../utils/pagination
 
 
-type
-  DataProc* = proc(receiverTimestamp: Timestamp, msg: WakuMessage, pubsubTopic: string) {.closure, raises: [Defect].}
+const 
+  STORE_DEFAULT_CAPACITY* = 25_000
+  STORE_MAX_OVERFLOW* = 1.3
+  STORE_DEFAULT_RETENTION_TIME* = chronos.days(30).seconds
+  STORE_MAX_PAGE_SIZE* = 100.uint64
+  STORE_MAX_TIME_VARIANCE* = getNanoSecondTime(20) # 20 seconds maximum allowable sender timestamp "drift" into the future
```

Comment:
There's no strong convention, and we've certainly been inconsistent here, but in general we avoid `CAPITAL_CASE` for `const`s following the Nim style guide: https://nim-lang.org/docs/nep1.html#introduction-naming-conventions

I personally prefer `PascalCase` (and I think this was the convention we converged on in nwaku code), but there is a case to be made for using `lowerCamel` (to avoid confusion with types). Also see this conversation:
https://github.com/status-im/nwaku/pull/941#discussion_r848477332

I'm OK if we extract "refactor constants to use consistent case" as a separate/follow-up task.

Reply:
Ok, I will change it 😁👍🏼